modules/quant_purger/README.md

```diff
@@ -1,8 +1,34 @@
 # Quant cache tag purger
 
-Adds a cache tag plugin which listen to invalidation events to queue Quant repopulation. This allows node_saves to update the main page but also trigger updates for pages that might be affected by the content update (eg. views pages). These will be added to the queue for the next core cron run to publish the updates.
+Adds a cache tag plugin which listens to Drupal invalidation events in order to
+queue Quant updates for related content.
+
+For example, this allows node edits to trigger the main (`/node`) page to update
+along with any other pages associated with the node through cache tags (e.g.
+views pages, taxonomy term pages, etc).
+
+This also works with other entities. For example, if a term is associated with
+several nodes, those nodes will be queued for updates when the term is edited.
+
+The associated content is added to the Quant queue and will be processed during
+the next core cron run. You can run the cron manually or wait for the site's
+cron to run on its regular schedule. For the latter, note that the static
+content will be out-of-sync with the Drupal site until the cron runs, which may
+cause confusion in some cases. Thus, it is recommended that cron is run right
```

Comment:
I would say that the recommendation is to have the quant queue run drush command running separately to core cron and having its frequency increased rather than running cron after a node save.

Reply:
Gotcha... could you revise how you'd like it?

Reply:
actually, I can revise

Reply:
@steveworley I changed it to:

```
To ensure that queued content is processed in a timely manner, you can set up a
Quant cron process that is separate from the core cron which just processes the
Quant queue. This Quant cron can be run more regularly than the core cron.
```

If you want any tweaks, feel free to suggest specific changes.